benchmark/Project.toml

```diff
@@ -0,0 +1,5 @@
+[deps]
+BenchmarkTools = "6e4b80f9-dd63-53aa-95a3-0cdb28fa8baf"
+ImageFiltering = "6a3955dd-da59-5b1f-98d4-e7296123deb5"
```

Comment:
```suggestion
```

As we are using `JULIA_LOAD_PATH='@:.:'`, I think `benchmark/Project.toml` should not include `ImageFiltering`. I think it's using the released version of `ImageFiltering` ATM.

(Alternatively, using `JULIA_LOAD_PATH='.:@:'` may be OK.)